gfreshii-webapp/src/main/java/com/edbootcamp/restManagers/RESTRecipeManagerImpl.java

```diff
@@ -0,0 +1,71 @@
+package com.edbootcamp.restManagers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.edbootcamp.api.manager.RecipeManager;
+import com.edbootcamp.api.views.RecipeView;
+import com.edbootcamp.view.RecipeViewImpl;
+
+@Service("recipeManager")
```

Comment:
I don't think you have conflicting implementations where you would need to specify this as a recipieManager,

Reply:
removed